sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/models/DeletedSecret.java

```diff
@@ -12,13 +12,13 @@
 
 /**
  * Deleted Secret is the resource consisting of name, recovery id, deleted date, scheduled purge date and its attributes
- * inherited from {@link SecretBase}.
+ * inherited from {@link SecretProperties}.
  * It is managed by Secret Service.
  *
  * @see SecretClient
  * @see SecretAsyncClient
  */
-public final class DeletedSecret extends SecretBase {
+public final class DeletedSecret extends Secret {
```

Comment:
extends from Secret or SecretProperties?

Reply:
Nope, 
Decided to have DeletedResource extend from Resource pattern, across all languages.

Reply:
So shouldn't the documentation too say, inherited from @Secret?

Reply:
Yup, updated.